src/lib.rs

```diff
@@ -75,9 +77,32 @@ pub fn sealed(_args: TokenStream, input: TokenStream) -> TokenStream {
     })
 }
 
+// generously offered by @danielhenrymantilla aka Yandros
+fn to_snake_case(s: &'_ str) -> String {
+    let mut ret = String::with_capacity(s.len());
+    let mut first = true;
+    s.bytes().for_each(|c| {
```

Comment:
Isn't `for c in s.chars() {` will be more obvoius here?

Since [`non_ascii_idents` are stabilized in 1.53](https://github.com/rust-lang/rust/pull/83799) it would be not great to break the expansion on them.

Reply:
Oh damn, non ascii idents change things indeed! While `.chars()` could work, I do imagine it won't work that well in practice, because correctly handling the casing of full unicode is subtle and non-trivial. At that point using `::heck` makes sense again, I'd say.

Reply:
I'll bring back `heck` until:
- Something breaks
- Someone complains 
Whichever comes first